src/realm/group_writer.cpp

```diff
@@ -132,21 +145,24 @@ GroupWriter::MapWindow::MapWindow(size_t alignment, util::File& f, ref_type star
 
 GroupWriter::MapWindow::~MapWindow()
 {
-    m_map.flush();
-    m_map.unmap();
+    unmap();
```

Comment:
Before to call `unmap` we should probably call also `msync()` ... just to be sure…